### PR TITLE
DIV-5521 Removed deprecated infra code / Prepare for prod AKS

### PR DESCRIPTION
**Pull request by hosainnet**
# Description

* Removed decpreated setVaultName
* Updated secrets to have vault na,e
* Updated java-logging to the latest version
* Updated the appinsights lib version
* Removed custom config in Dockerfile
* Use hmctspublic docker repo

Fixes DIV-5521

## Type of change

- [x] New feature (non-breaking change which adds functionality)

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: Jenkinsfile_CNP [hunk 1]
```diff
@@ -59,8 +57,4 @@ withPipeline(type, product, component) {
     after('functionalTest:preview') {
         steps.archiveArtifacts allowEmptyArchive: true, artifacts: '**/site/serenity/**/*'
     }
-
-    onMaster {
-        enableSlackNotifications(channel)
-    }
```
Comment (hosainnet) [inline Jenkinsfile_CNP]: It's enabled by default

File: Dockerfile [hunk 2]
```diff
@@ -1,14 +1,11 @@
-FROM hmcts/cnp-java-base:openjdk-8u181-jre-alpine3.8-1.0
+ARG APP_INSIGHTS_AGENT_VERSION=2.3.1
+FROM hmctspublic.azurecr.io/base/java:openjdk-8-distroless-1.0
```
Comment (hosainnet) [inline Dockerfile]: Updated to the public registry

File: Dockerfile [hunk 3]
```diff
@@ -1,14 +1,11 @@
...
-ENV APPLICATION_TOTAL_MEMORY 1024M
-ENV APPLICATION_SIZE_ON_DISK_IN_MB 47
 
+COPY lib/applicationinsights-agent-2.3.1.jar lib/AI-Agent.xml /opt/app/
 COPY build/libs/$APP /opt/app/
 
-WORKDIR /opt/app
-
-HEALTHCHECK --interval=100s --timeout=100s --retries=10 CMD http_proxy="" wget -q http://localhost:4006/health || exit 1
-
```
Comment (hosainnet) [inline Dockerfile]: Removed as suggested by CNP

File: Jenkinsfile_CNP [hunk 4]
```diff
@@ -17,12 +17,12 @@ def type = "java"
 def product = "div"
 def component = "emca"
 
-def channel = '#div-dev'
-
-List<LinkedHashMap<String, Object>> secrets = [
-        secret("div-doc-s2s-auth-secret", "IDAM_AUTH_SECRET"),
-        secret("idam-secret", "AUTH_IDAM_CLIENT_SECRET"),
-        secret("div-doc-s2s-auth-secret", "AUTH_PROVIDER_SERVICE_CLIENT_KEY")
```
Comment (hosainnet) [inline Jenkinsfile_CNP]: Deprecated way of loading secrets

File: build.gradle [hunk 5]
```diff
@@ -43,7 +43,7 @@ def versions = [
         reformHealth                            : '0.0.3',
         reformPropertiesVolume                  : '0.0.4',
         reformsHttpProxySpringBootAutoconfigure : '1.1.0',
-        reformsJavaLogging                      : '4.0.1',
```
Comment (hosainnet) [inline build.gradle]: Updated to the latest, which also required the new appinsights agent jar below

File: Dockerfile (hunk 3 above)
Comment (timja) [reply Dockerfile]: CNP is gone, long live platform

File: Dockerfile [hunk 6]
```diff
@@ -1,14 +1,11 @@
-FROM hmcts/cnp-java-base:openjdk-8u181-jre-alpine3.8-1.0
+ARG APP_INSIGHTS_AGENT_VERSION=2.3.1
```
Comment (qzhou-hmcts) [inline Dockerfile]: we probably need to update or remove this https://github.com/hmcts/div-evidence-management-client-api/tree/master/lib

Comment (hosainnet) [reply Dockerfile]: Already done below..

**Pull request merged**